disnake/ext/commands/base_core.py

```diff
@@ -215,6 +215,14 @@ def _ensure_assignment_on_copy(self, other: AppCommandT) -> AppCommandT:
             # _max_concurrency won't be None at this point
             other._max_concurrency = cast(MaxConcurrency, self._max_concurrency).copy()
 
+        if self.body._default_member_permissions != other.body._default_member_permissions:
+            if "default_member_permissions" not in other.__original_kwargs__:
+                other.body._default_member_permissions = self.body._default_member_permissions
+            elif self.body._default_member_permissions is not None:
+                raise ValueError(
+                    "Cannot set `default_member_permissions` in both decorator and slash_command_attrs"
```

Comment:
this should tell the user how they can be set instead (via direct arguments) although I think this is bad design and we should try to get rid of this error altogether

Reply:
Should I change the error message or remove it and make decorators overwrite <...>_command_attrs?

Reply:
I'd personally go with overwriting the cog-wide attrs, as that's how it works everywhere else, I believe.

Reply:
I'd also prefer this approach, I should probably ask @shiftinv as well

Reply:
Yup, I agree with overwriting cog-wide attributes, that's the way others are handled as well.